docs/index.asciidoc

```diff
@@ -1,10 +1,14 @@
 [[logstash-reference]]
 = Reference
 
-:branch:          1.5
-:logstash_version: 1.5.0
+:branch:                1.5
+:logstash_version:      1.5.0
```

Comment:
Since @suyograo didn't change it when he updated this index doc earlier today, I figured he had a reason and decided to leave it as is.